Formula/sphinx.rb

```diff
@@ -1,25 +1,25 @@
 class Sphinx < Formula
-  desc "Sphinx is a full-text search engine"
+  desc "Full-text search engine"
   homepage "http://www.sphinxsearch.com"
 
-  stable do
-    url "http://sphinxsearch.com/files/sphinx-2.2.10-release.tar.gz"
-    sha256 '054cb86e64bd48997d89386e1224d4405063d9857b2d7c33dc6cc1b9ef6df440'
-  end
+  head "https://github.com/sphinxsearch/sphinx.git"
 
-  devel do
-    url "http://sphinxsearch.com/files/sphinx-2.3.1-beta.tar.gz"
-    sha256 "0e5ebee66fe5b83dd8cbdebffd236dcd7cd33a7633c2e30b23330c65c61ee0e3"
+  stable do
```

Comment:
Can drop this entire block since we're not manipulating it. Safe to revert to the:
```ruby
desc "Full-text search engine"
homepage "http://www.sphinxsearch.com"
url "http://sphinxsearch.com/files/sphinx-2.2.11-release.tar.gz"
sha256 "6662039f093314f896950519fa781bc87610f926f64b3d349229002f06ac41a9"
```
Format.